src/test/java/net/smartcosmos/usermanagement/role/persistence/RolePersistenceServiceTest.java

```diff
@@ -129,6 +129,58 @@ public void thatUpdateRoleSucceeds() {
     }
 
     @Test
+    public void thatUpdateRoleSucceedsWithSameName() {
```

Comment:
Hmm, I partly agree, because in `thatUpdateRoleAddingAuthoritySucceeds` I wouldn't include the name in the request.

I particularly tested for the same name in the request, because these updates were blocked previously. That's why I'm actually not interested in the authorities, but in the name not getting rejected.